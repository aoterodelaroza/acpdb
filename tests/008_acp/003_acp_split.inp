## check: 003_acp_split.out -a1e-10
## delete: 003_acp_split_a-1.acp
## delete: 003_acp_split_a-2.acp
## delete: 003_acp_split_b-1.acp
## delete: 003_acp_split_b-2.acp
## labels: regression quick

acp load bleh
 C l 0.2 40.0
 N l 0.2 42.0
end
acp split bleh 003_acp_split_a
acp split bleh 003_acp_split_b coef 1.0

