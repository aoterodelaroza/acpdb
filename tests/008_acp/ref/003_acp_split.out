* ACP LOAD bleh

* ACP SPLIT bleh creates files 003_acp_split_a-*.acp

* ACP SPLIT bleh creates files 003_acp_split_b-*.acp

